pyfr/solvers/baseadvec/inters.py

```diff
@@ -123,8 +123,8 @@ def _exp_opts(self, opts, lhs, default=None):
 
             # Substitute variables
             ex = re.sub(r'\b({0})\b'.format('|'.join(subs)),
-                        lambda m: subs[m.group(1)], ex)
-            exprs[k] = ex
+                        lambda m: '('+subs[m.group(1)]+')', ex)
```

Comment:
Yes, it will expand out to either a value like `3.14` or another variable like `ploc[1]`.  Variable to variable and variable to number are both safe in the sense that they do not change the precedence.